### ISSUE TITLE
UI Accessibility work

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3391]
Bug 402111: Accessibility: MAS42A : High Contrast: Environment selection from combo box is not in compliance with HC themes

Bug 402112: Accessibility: MAS42A : High Contrast: Select Environment combo box when focused using keyboard doesn't get a highlight

Bug 402121: Accessibility: MAS42A : High Contrast: Text selection in Workspaces pane > Text boxes in not in compliance with HC themes

Bug 402675: Accessibility: MAS42A: High Contrast: Load package radio buttons when focused using keyboard are hardly identifiable in HC themes

Bug 402676: Accessibility: MAS42A: High Contrast: Uninstall package buttons when focused using keyboard are hardly identifiable in HC themes

Bug 402678: Accessibility: MAS42A: High Contrast: Vertical separator/Grid Splitter when focused using keyboard is not identifiable in HC black

Bug 403504: Accessibility: MAS42A: High Contrast: Workspaces > Local Connection Settings > Cancel button is not in compliance with HC themes

Bug 403509: Accessibility: MAS42A: High Contrast: 'Dismiss' and 'Dismiss All' buttons in error pop-up bar are not in compliance with HC themes

**Comment by AlexanderSher** [comment]
Added 2 more

**Comment by MikhailArkhipov** [comment]
403509 and 403504 - we can just make them into buttons and be done.

**Comment by AlexanderSher** [comment]
403509 will be fixed as a result of #3458
